src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react'
+import {useAuth0} from '@auth0/auth0-react'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import CardContent from '@mui/material/CardContent'
+import CardHeader from '@mui/material/CardHeader'
+import InputBase from '@mui/material/InputBase'
+import Paper from '@mui/material/Paper'
+import useTheme from '@mui/styles/useTheme'
+import {TooltipIconButton} from '../Buttons'
+import useStore from '../../store/useStore'
+import {postIssue} from '../../utils/GitHub'
+import Submit from '../../assets/icons/Submit.svg'
+
+
+/**
+ * Note card create
+ *
+ * @param {string} username
+ * @param {string} avatarUrl
+ * @return {object} React component
+ */
+export default function NoteCardCreate({
+  username = '',
+  avatarUrl = '',
+}) {
+  const repository = useStore((state) => state.repository)
+  const toggleIsCreateNoteActive = useStore((state) => state.toggleIsCreateNoteActive)
+  const [title, setTitle] = useState('')
+  const [body, setBody] = useState('')
+  const {user, isAuthenticated} = useAuth0()
+  const accessToken = useStore((state) => state.accessToken)
+  const createdNotes = useStore((state) => state.createdNotes)
+  const setCreatedNotes = useStore((state) => state.setCreatedNotes)
+  const theme = useTheme()
+
+
+  /**
+   * create issue takes in the title and body of the note
+   *
+   */
```

Comment:
`@return` jsdoc

Reply:
?

Reply:
there is no return here, I think.

Reply:
https://stackoverflow.com/questions/4759175/how-to-return-void-in-jsdoc